README.md

```diff
@@ -124,6 +124,19 @@ $ curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/ni
 $ NIX_BUILD_GROUP_NAME=nixbuilder ./nix-installer install linux-multi --nix-build-group-id 4000
 ```
 
+## Upgrading Nix
+
+You can upgrade Nix (to the version specified [here](https://raw.githubusercontent.com/NixOS/nixpkgs/master/nixos/modules/installer/tools/nix-fallback-paths.nix)) by running:
+
+```
+nix upgrade-nix
+```
+
+> **Note**
+> Might require sudo (e.g. on **MacOS**).
+
+Alternatively, you can [uninstall](#uninstalling) and [reinstall](#usage).
```

Comment:
This alternative installs version `2.15.0` instead of `2.15.1` that is specified at https://raw.githubusercontent.com/NixOS/nixpkgs/master/nixos/modules/installer/tools/nix-fallback-paths.nix.

I grepped the repository searching for a reference for it but I do not understand why running the installer installs `2.15.0` instead of `2.15.1`.

Reply:
That's because we pinned to 2.15.0 by default here: https://github.com/DeterminateSystems/nix-installer/blob/55cbf41a007aa0f230bb3744c34867b460be4a0a/src/settings.rs#L13

We've since merged https://github.com/DeterminateSystems/nix-installer/pull/597 which updates us to 2.17.0, but that hasn't yet made it into a release.

Reply:
We should cut a release soon!

Reply:
I grepped the wrong version (2.5.0 instead of 2.15.0) 🤦. Thanks for explaining @cole-h